assets/src/editorIndex.js

```diff
@@ -24,32 +24,37 @@ import { SubPagesBlock } from './blocks/SubPages/SubPagesBlock';
 import { blockEditorValidation } from './BlockEditorValidation';
 import { ENFormBlock } from './blocks/OldENForm/ENFormBlock';
 import { registerGuestBookBlock } from './blocks/GuestBook/GuestBookBlock';
+import { HubspotFormBlock } from './blocks/HubspotForm/HubspotFormBlock';
 
-blockEditorValidation();
-new ArticlesBlock();
-new CarouselHeaderBlock();
-registerColumnsBlock();
-new CookiesBlock();
-new CounterBlock();
-new CoversBlock();
-new GalleryBlock();
-new HappypointBlock();
-registerMediaBlock();
-registerSocialMediaBlock();
-new SocialMediaCardsBlock();
-registerSplittwocolumnsBlock();
-registerSpreadsheetBlock();
-registerSubmenuBlock();
-new SubPagesBlock();
-new TakeactionboxoutBlock();
-new ENFormBlock();
-registerTimelineBlock();
-registerGuestBookBlock();
-
-addBlockFilters();
-setupImageBlockExtension();
-addButtonLinkPasteWarning();
-replaceTaxonomyTermSelectors();
-setupCustomSidebar();
-setUpCssVariables();
-blockEditorValidation();
+try {
```

Comment:
Why do we need to wrap this in a try-catch block?

Reply:
It was suggested by SonarCloud

Reply:
I think it only tests PRs, so probably it's not historically available for previous commits.